### ISSUE TITLE
MysticalWildlife defines entries on classes that it does not own

### ISSUE THREAD

**Issue by LuminescentMoon** [Lykrast/MysticalWildlife#6]
This can cause crashes that only appear on a client connected to a dedicated server (not integrated).

Log:
```
defineId called for: class net.minecraft.entity.EntityAgeable from class lykrast.mysticalwildlife.common.entity.EntityFurzard
defineId called for: class net.minecraft.entity.EntityAgeable from class lykrast.mysticalwildlife.common.entity.EntityYagaHog
```

See https://github.com/MinecraftForge/MinecraftForge/pull/5129 for more details about this issue.

This issue is that [EntityDataManager is used](https://github.com/Lykrast/MysticalWildlife/blob/8b73bbc724d66b114e2d79dd0357e2a417f439a4/src/main/java/lykrast/mysticalwildlife/common/entity/EntityYagaHog.java#L51v) for tracking entity data. Instead, the modern approach is to use Forge [Capabilities](https://mcforge.readthedocs.io/en/latest/datastorage/capabilities/). Here's a handy tutorial on how to implement them [here](https://www.planetminecraft.com/blog/forge-tutorial-capability-system/).

**Comment by Lykrast** [comment]
Yeah that's an oversight. It's fixed on the 1.14 version so I'll make a small update to the 1.12 to fix that then.

**Comment by Lykrast** [comment]
Fixed for next 1.12 update (1.3.0.1) which has been sent to approval on cf.
